### PR TITLE
Update jest config

### PR DESCRIPTION
**Pull request by davidswinegar**
Add this line, which is required to get internal CI to pass so that
we can add this worker to the open-source version.

### CODE REVIEW

File: jest/jest.config.js [hunk 1]
```diff
@@ -25,6 +25,7 @@ module.exports = {
   ],
   setupTestFrameworkScriptFile: "<rootDir>/packages/webviz-core/src/test/setupTestFramework.js",
   moduleNameMapper: {
+    "worker-loader!./ChartJSWorker": "<rootDir>/webviz-core/src/test/MockWorker.js",
```
Comment (janpaul123) [inline jest/jest.config.js]: Should we do this by default for `worker-loader!*`?

Comment (jtbandes) [reply jest/jest.config.js]: Why is this path different from the others around it? (missing `/packages/`)

Comment (janpaul123) [reply jest/jest.config.js]: Ah yeah, didn't notice that; that seems wrong! 😮

Comment (davidswinegar) [reply jest/jest.config.js]: Whoops, didn't notice these comments before merging. Fixed with https://github.com/cruise-automation/webviz/pull/343

**Pull request merged**